coderd/coderdtest/coderdtest.go

```diff
@@ -264,8 +264,10 @@ func NewOptions(t testing.TB, options *Options) (func(http.Handler), context.Can
 	if options.DeploymentValues == nil {
 		options.DeploymentValues = DeploymentValues(t)
 	}
-	// This value is not safe to run in parallel. Force it to be false.
-	options.DeploymentValues.DisableOwnerWorkspaceExec = false
+	// This value is not safe to run in parallel.
+	if options.DeploymentValues.DisableOwnerWorkspaceExec {
+		t.Logf("WARNING: DisableOwnerWorkspaceExec is set, this is not safe in parallel tests!")
```

Comment:
It'd be cool if we could actually check if parallel has been set (or will be set!), maybe we should implement out own `testing.TB`... 😆

Reply:
We could possibly set a dummy environment variable; this should cause `t.Parallel()` to panic.